docker-compose.yml

```diff
@@ -1,5 +1,3 @@
-version: '3'
```

Comment:
This key is just information and can't be ommitted, it's remove a warning during build process.
More Info: https://github.com/compose-spec/compose-spec/blob/master/spec.md#version-and-name-top-level-elements

Reply:
I think this change is not strictly required for fixing the current issue. And I already experienced issue when version is missing. Please keep it.